product_docs/docs/pem/9/troubleshooting.mdx

```diff
@@ -88,3 +88,32 @@ In some situations, you might need to uninstall the PEM server, reinstall it, an
     ```shell
     /usr/edb/pem/bin/configure-pem-server.sh
     ```
+
+## Updating pemagent service file values if agent configuration file path is modified
```

Comment:
I think we can create separate child topics for the server and agent under the troubleshooting topic. Also, the heading for this need to address the problem and not the solution.

Reply:
Based on the last discussion with Nidhi, two child topics, one for the server and another for the agent, are not required right now in the "Troubleshooting" topic. 
I have modified the heading to highlight the problem: "Agent status on the PEM dashboard is displayed as unknown"